controllers/listbuilder/admin/categories/CategoryListbuilderHandler.inc.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @file controllers/listbuilder/admin/categories/CategoryListbuilderHandler.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2000-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class CategoryListbuilderHandler
+ * @ingroup controllers_listbuilder_admin_categories
+ *
+ * @brief Listbuilder for managing journal categories.
+ */
+
+import('lib.pkp.classes.controllers.listbuilder.ListbuilderHandler');
+
+class CategoryListbuilderHandler extends ListbuilderHandler {
+
+	/**
+	 * Constructor
+	 */
+	function CategoryListbuilderHandler() {
+		parent::ListbuilderHandler();
+		$this->addRoleAssignment(
+			ROLE_ID_SITE_ADMIN,
```

Comment:
No -- this grid is to maintain the list of available categories; there's another listbuilder to assign them to journals, which is the JM's purview.